src/main/kotlin/no/nav/syfo/job/VarselSender.kt

```diff
@@ -112,4 +111,18 @@ class VarselSender(
             }
         }
     }
+
+    fun getAllUnsendMerVeiledningVarslerLastMonth(): List<PPlanlagtVarsel> {
```

Comment:
Her skal det være `t` i stedet for `d` i `unsend`

Reply:
Jeg synes det er litt synd at det har havnet mye merveiledning-spesifikk kode i denne klassen som egentlig skal være generell. Man kan f.eks. se for seg at man innkapsulerer det i en egen klasse/tjeneste som håndterer "mer veiledning", men ettersom vi på sikt kun får planlegging av mer veiledning, og aktivitetskravvarsler skal bort, så tenker jeg at det er greit.

Reply:
Jeg har tenkt litt mer på dette, og hvis man hadde gått videre med å lage en egen tjeneste for dette kunne man latt tjenesten håndtere alt som bestemmer hvilke mer veiledning varsler som skal sendes i dag:
- Hente alle planlagte varsler fra planlagt_varsel-tabellen
- Hente alle planlagte varsler fra max dato-tabellen
- Hente utsendte mer veiledning den siste måneden
- Sjekke om personen har aktiv sykmelding på varseldatoen

Da ville også testene blitt mye mer ryddig, og man får ikke mer veiledning-spesifikke tester i `SendVarselServiceSpek` og `VarselSenderSpek`

Jeg synes vi skal gjøre dette, men det kan kanskje være greit å ta det i en egen PR

Reply:
Da ville det kanskje blitt noe slikt (i pseudo-kode)
```suggestion
  var varslerToSendTodayAktivitetskrav = databaseAccess.fetchPlanlagtVarselByTypeAndUtsendingsdato(AKTIVITETSKRAV, LocalDate.now())
  var varslerToSendTodayMerVeiledning = merVeiledningService.finnMerVeiledningVarslerSomSkalSendes()
  var varslerToSendToday = merge(varslerToSendTodayAktivitetskrav, varslerToSendTodayMerVeiledning)
```

Reply:
Ooops, de siste kommentarene gjelder egentlig linje 26 i VarselSender